.github/workflows/ci-testing-pull-request.yml

```diff
@@ -10,6 +10,20 @@ on:
     branches:
       - "master"
 
+  workflow_dispatch:
+    inputs:
+      target_repo:
+        description: full repository name (e.g. 'ITISFoundation/osparc-simcore') to check backwards compatibility against
+        required: true
+        default: "ITISFoundation/osparc-simcore"
+        type: environment
+      target_branch:
+        description: branch name (e.g. 'master') to check backwards compatibility against
```

Comment:
against ... ?

Reply:
Check backwards compatibility against `target_repo` which is the full repository name (e.g. 'ITISFoundation/osparc-simcore') and `target_branch`

?